src/Annotation/Parser/OptionsParserBot.php

```diff
@@ -0,0 +1,342 @@
+<?php
+
+/**
+ * Represents the state of the cursor during parsing of annotation options
+ */
+
+namespace Framework3\Annotation\Parser;
+
+class OptionsParserBot extends AbstractParserBot
```

Comment:
Codacy found an issue: [The class OptionsParserBot has 12 public methods. Consider refactoring OptionsParserBot to keep number of public methods under 10.](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)